queries.sql

```diff
@@ -0,0 +1,19 @@
+/* Fix typo */
+vet_clinic=# UPDATE animals
+             SET name = 'Gabumon' WHERE name = 'GABUMON';
+             
```

Comment:
- [ ] Kindly add this query in the **data.sql** file because **data.sql** file is responsible for holding those queries which manipulate the data in the database :thinking: and **queries.sql** is responsible for holding those queries which retrieve data from the database :+1: